<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoFrames</class>
 <widget class="QWidget" name="VideoFrames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoFrames</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#VideoFrames {
    background-color: #FFFFFF;
    border-radius: 8px;
    border: 1px solid #E2E8F0;
    margin-top: 20px;
    margin-bottom: 20px;
}

QLabel#titleLabel {
    font-size: 20px;
    font-weight: bold;
    color: #1E293B;
    background-color: transparent;
    margin-top: 20px;
}

QScrollArea {
    border: none;
    background-color: transparent;
    min-height: 350px;
    max-height: 350px;
}

QScrollBar:horizontal {
    border: none;
    background: #F1F5F9;
    height: 8px;
    border-radius: 4px;
}

QScrollBar::handle:horizontal {
    background: #CBD5E1;
    min-width: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}

QFrame.frameItem {
    min-height: 300px;
    max-height: 300px;
    background-color: #FFFFFF;
    border: 2px solid #E2E8F0;
    border-radius: 6px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Các khung hình từ video</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="framesScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>758</width>
        <height>350</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>350</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QFrame" name="frame1">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="class" stdset="0">
          <string>frameItem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="class" stdset="0">
          <string>frameItem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="class" stdset="0">
          <string>frameItem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame4">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="class" stdset="0">
          <string>frameItem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame5">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="class" stdset="0">
          <string>frameItem</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>